frontend/components/songs/song_card.jsx

```diff
@@ -9,9 +10,13 @@ const SongCard = ({
     song,
     history,
     index,
+    songCardDropdownState,
+    openSongCardDropdown,
+    closeSongCardDropdown,
 }) => {
 
-    const { title,
+    const { id,
```

Comment:
nit: small nit-pick on formatting, should this be on a new line?
```suggestion
    const { 
        id,
```